<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoforBilgisiVeriGirisFormu</class>
 <widget class="QWidget" name="SoforBilgisiVeriGirisFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Şoför Bilgisi</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Kaynaklar.qrc">
    <normaloff>:/Resimler/Resimler/sofor.png</normaloff>:/Resimler/Resimler/sofor.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QLabel" name="lbTCKimlikNo">
           <property name="text">
            <string>TCKimlik No</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="leAdi"/>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="leTCKimlikNo"/>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="leHESKodu"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="lbTecrubesi">
           <property name="text">
            <string>Tecrübesi</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="leTecrubesi"/>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="leEhliyetSinifi"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="lbSoayadi">
           <property name="text">
            <string>Soyadı</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbHesKodu">
           <property name="text">
            <string>Hes Kodu</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="lbAdi">
           <property name="text">
            <string>Adı</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="leSoyadi"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="lbEhliyetSinifi">
           <property name="text">
            <string>Ehliyet Sınıfı</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="lbCinsiyet">
           <property name="text">
            <string>Cinsiyet</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QRadioButton" name="rbErkek">
             <property name="text">
              <string>Erkek</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbKadin">
             <property name="text">
              <string>Kadın</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pbKaydet">
           <property name="text">
            <string>Kaydet</string>
           </property>
           <property name="icon">
            <iconset resource="../../Kaynaklar.qrc">
             <normaloff>:/Resimler/Resimler/kaydet.png</normaloff>:/Resimler/Resimler/kaydet.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbIptal">
           <property name="text">
            <string>İptal</string>
           </property>
           <property name="icon">
            <iconset resource="../../Kaynaklar.qrc">
             <normaloff>:/Resimler/Resimler/iptal.png</normaloff>:/Resimler/Resimler/iptal.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lbIcon">
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../Kaynaklar.qrc">:/Resimler/Resimler/sofor.png</pixmap>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="lbSoforBilgisi">
         <property name="text">
          <string>Şoför Bilgisi</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leAdi</tabstop>
  <tabstop>leSoyadi</tabstop>
  <tabstop>leTCKimlikNo</tabstop>
  <tabstop>leHESKodu</tabstop>
  <tabstop>rbErkek</tabstop>
  <tabstop>rbKadin</tabstop>
  <tabstop>leEhliyetSinifi</tabstop>
  <tabstop>leTecrubesi</tabstop>
  <tabstop>pbKaydet</tabstop>
  <tabstop>pbIptal</tabstop>
 </tabstops>
 <resources>
  <include location="../../Kaynaklar.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pbIptal</sender>
   <signal>clicked()</signal>
   <receiver>SoforBilgisiVeriGirisFormu</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
